src/providers/VoiceFocusProvider/docs/VoiceFocusProvider.stories.mdx

```diff
@@ -0,0 +1,105 @@
+<Meta title="SDK Providers/VoiceFocusProvider" />
+
+# VoiceFocusProvider
+
+The `VoiceFocusProvider` provides a function transforming a normal device to an Amazon voice focus device, and the support state of voice focus status.
+
+This provider is independent from `MeetingProvider`. You can put `VoiceFocusProvider` wherever you want in the component tree to control the workflow of voice focus.
+
+## State
+
+```typescript
+{
+  // Whether the voice focus is supported. The default value is undefined, then changes to true or false.
+  isVoiceFocusSupported: boolean | undefined;
+
+  // A function to transform an audio input device to an Amazon voice focus device
+  addVoiceFocus: (device: Device) => Promise<Device | VoiceFocusTransformDevice>;
+}
+
+```
+
+You can access the state by using the [useVoiceFocus](/docs/sdk-hooks-usevoicefocus--page) hook.
+
+## Importing
+
+```javascript
+import { VoiceFocusProvider } from 'amazon-chime-sdk-component-library-react';
+```
+
+## Usage
+
+```jsx
+import React from 'react';
+import {
+  MeetingProvider,
+  VoiceFocusProvider,
+  useMeetingManager,
+  useVoiceFocus,
+} from 'amazon-chime-sdk-component-library-react';
+
+const App = () => (
+  <VoiceFocusProvider>
+    <MeetingProvider>
+        <MyChild />
+    </MeetingProvider>
+  </VoiceFocusProvider>
+);
+
+const MyChild = () => {
+  const meetingManager = useMeetingManager();
+  const [isVoiceFocusOn, setIsVoiceFocusOn] = useState(false);
+  const [isVoiceFocusEnabled, setIsVoiceFocusEnabled] = useState(false);
+  const [device, setDevice] = useState(meetingManager.selectedAudioInputTransformDevice)
+  const {isVoiceFocusSupported, transformToVoiceFocus} = useVoiceFocus();
+
+  useEffect(() => {
+    meetingManager.subscribeToSelectedAudioInputTransformDevice(setDevice);
+    return (): void => {
```

Comment:
We can remove `: void` since we mention this is `jsx` block. Sorry, I do not know and remember whether storybook supports `tsx`.

Reply:
removed `: void`